README.md

```diff
@@ -236,46 +255,82 @@ The returned array contains objects of type `StreemCallLogEntry`, which have the
     endDate: Date?             // Session end time.
     participants: [StreemUser] // Session participants. One for an Onsite Streem, two for a two-way Streem.
     streemshotsCount: Int      // The number of Streemshots captured during the session.
+    hasMesh: Bool              // Whether the streem has a mesh with it or not.
 ```
 
 ### Displaying and Editing Streemshots
```

Comment:
`Displaying and Editing Streemshots` -> `Displaying and Editing Artifacts`

Reply:
👍